sv_inline.h

```diff
@@ -0,0 +1,527 @@
+/*    sv.h
+ *
+ *    Copyright (C) 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000,
+ *    2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009 by Larry Wall and others
+ *
+ *    You may distribute under the terms of either the GNU General Public
+ *    License or the Artistic License, as specified in the README file.
+ *
+ */
+
+
+/* 2022 */
+/* BLAH BLAH BLAH */
```

Comment:
You might want to remove this :)

Reply:
Oh yes, that was my placeholder 'cos I wasn't sure what copyright message and descriptive text to include.

I'll change it to the following, unless there are any other recommendations for what else to mention.

```
/*    sv_inline.h
 *
 *    Copyright (C) 2022 by Larry Wall and others
 *
 *    You may distribute under the terms of either the GNU General Public
 *    License or the Artistic License, as specified in the README file.
 *
 */

/* This file contains the newSV_type and newSV_type_mortal functions, as well as
 * the various struct and macro definitions they require. In the main, these
 * definitions were moved from sv.c, where many of them continue to also be used.
 */
```

Reply:
I've pushed that plus an additional chunk of notes & rationale.